playground/mocks/data/idp/idx/enroll-profile-new.json

```diff
@@ -20,18 +20,27 @@
               "value": [
                 {
                   "name": "lastName",
+                  "type": "string",
```

Comment:
Just curious, did something force you to add `"type": "string"`? IIRC It's implied according to the Ion Spec.

Reply:
I have `type ==== 'string'` as part of my test logic before I can cast the field value to a `string` and compare its `length` to `maxLength` and `minLength`, so I had to add this to the mock otherwise the error isn't thrown

Reply:
@trevoring-okta does the real API response send this too?

Reply:
@shawyu-okta I believe so, here's an example response for the email field that my preview org sends me:
`label
: 
"Email"
maxLength
: 
100
name
: 
"userProfile.email"
required
: 
true
type
: 
"string"`

Reply:
Is it possible that not all orgs send the `type`?

Reply:
I think all orgs and endpoints should send `type` then because I imagine they should use the same serializer for Ion Spec. I guess we could be even safer by assuming `type === 'string'` iff `type` is `undefined`.